core/domain/wipeout_service.py

```diff
@@ -179,7 +208,10 @@ def pre_delete_user(user_id):
         date_now - PERIOD_AFTER_WHICH_USERNAME_CANNOT_BE_REUSED)
     normalized_long_term_username = (
         user_settings.normalized_username
-        if user_settings.created_on < date_before_which_username_should_be_saved
+        if (
+           user_settings.created_on is not None and
```

Comment:
I think in the previous code an error would be thrown if it's None.

Shouldn't we replicate that behaviour here?

Reply:
Done, an exception is added docstring is also updated!